ai_agents/ai_travel_planning_agent/tools.py

```
@@ -0,0 +1,71 @@
+            search_depth="advanced",
+            max_results=5,
+            include_answer=True,
+        )
+        parts = []
+        if response.get("answer"):
+            parts.append(f"Summary: {response['answer']}\n")
+        for r in response.get("results", []):
+            snippet = r.get("content", "")[:600].strip()
+            parts.append(f"**{r['title']}**\nURL: {r['url']}\n{snippet}")
+        return "\n\n---\n\n".join(parts) if parts else "No results found."
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)